src/processes/helper/addUserToTopicChannel.js

```diff
@@ -1,41 +1,39 @@
-const { convertString, ICON_TOPIC_CHANNEL, CHANNEL_TYPE_TOPIC } = require('../../utils');
-const { LogError } = require('../../databases/models');
-const StreamChat = require("stream-chat").StreamChat;
+const {convertString, ICON_TOPIC_CHANNEL, CHANNEL_TYPE_TOPIC} = require('../../utils');
+const {LogError} = require('../../databases/models');
+const StreamChat = require('stream-chat').StreamChat;
 
 const addUserToTopicChannel = async (user_id, topics) => {
-    try {
-        let userId = user_id.toString();
-        const serverClient = StreamChat.getInstance(
-            process.env.API_KEY,
-            process.env.SECRET
-        );
-        topics.map(async (item) => {
-            serverClient.createToken(userId);
-            const channelId = item;
-            const members = [];
-            members.push(userId);
+  try {
+    let userId = user_id.toString();
+    const serverClient = StreamChat.getInstance(process.env.API_KEY, process.env.SECRET);
+    topics.map(async (topic) => {
+      const newTopic = convertString(topic, '-', '').toLowerCase();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The variable `newTopic` is created for each topic in the loop but it's not clear why the original `topic` variable isn't used directly. If there's no specific reason for this, consider removing the `newTopic` variable to reduce complexity.

```diff
-      const newTopic = convertString(topic, '-', '').toLowerCase();
+      topic = convertString(topic, '-', '').toLowerCase();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->